test/vagrant-service-manager/binary_handlers/adb_kubernetes_binary_handler_test.rb

```diff
@@ -0,0 +1,103 @@
+require_relative '../../test_helper'
+require 'vagrant/util/downloader'
+
+# Tests through ADBKubernetesBinaryHandler to BinaryHandler
+module VagrantPlugins
+  module ServiceManager
+    describe ADBKubernetesBinaryHandler do
+      before do
+        @machine = fake_machine
+        @options = { type: :kubernetes, '--cli-version' => '1.2.0' } # 1.2.0 is available in CDK/ADB VM
```

Comment:
Should 1.2.0 be hardcoded here?  I believe this information is somewhere else too.  I am cool with having an issue opened that notes we need to re-factor this, it appropriate so as to not hold this PR up.